views/elements/current_and_next.tmpl

```diff
@@ -0,0 +1,20 @@
+{{define "current_and_next"}}
```

Comment:
What's your reasoning for making this a separate file? It leaves index.tmpl rather small